### ISSUE TITLE
NVDA is announcing Read only field 'Look for requirement group' as editable in browse mode.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14827]
### Steps to reproduce:
1. Launch appropriate screen reader NVDA.
2. Open above URL https://codepen.io/phtucker/pen/Jjaampp in chrome browser.
3.Default focus would be inside HTML code.
4.Press Esc key to come out of it.
5.Press shortcut key B to navigate to 'Console' button.
6.Press Up arrow key to navigate to 'Look for requirement group' button and observe the issue, if the screen reader is announcing the field as editable.

### Actual behavior:
While navigating in browse mode using arrow keys to the read only field, screen reader NVDA is announcing the read only “Look for requirement group” field as editable.

### Expected behavior:
While navigating in browse mode using arrow keys to the read only field, screen reader NVDA should be announcing the “Look for requirement group” field as read only.

### NVDA logs, crash dumps and other attachments:
<img width="955" alt="205471" src="https://user-images.githubusercontent.com/93735775/231692487-e2a2b8a9-5fac-46ad-a1dd-2015b63e2f43.png">

https://user-images.githubusercontent.com/93735775/231692583-d4c7ab3e-432b-46c3-bbbf-b32650a89f7c.mp4

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
Version 2023.1
#### Windows version:
Version 22H2(Os build 22621.1555)
Chrome Version 112.0.5615.50 (Official Build) (64-bit)
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, issue existing in NVDA version 2022.4
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes, have tried that and still issue is existing.

**Comment by Adriani90** [comment]
This impacts all types of controls other than edit fields or input text elements. I am closing this as duplicate of #13672 since this needs to be solved in NVDA from a global perspective for all readOnly state no matter which type of element it is.
